docs/apis/avalanchego/apis/info.md

```diff
@@ -128,7 +128,13 @@ Get the ID of this node.
 #### **Signature**
 
 ```sh
-info.getNodeID() -> {nodeID: string}
+info.getNodeID() -> {
+    nodeID: string,
+    nodePOP: {
+        publicKey: string,
+        proofOfPossession: string
+    }
```

Comment:
We should add info for each returned data. Previously nodeID by itself is self-explanatory.